tests/virtual_machines_test.go

```diff
@@ -100,6 +100,30 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 				}
 			})
 
+			Context("and the client tries to assign the same MAC address for two different vm, one created with dryRun=All mode", func() {
+				It("should success creating a vm with the same mac allocated to a dry run", func() {
+					By("Creating a vm with dry run mode")
```

Comment:
will move the creation of the vm dryRun to `BeforeEach()`
The test should be about the pool allocation and not the dryRun creation